source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,106 @@
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can remove documents by passing a query filter into the
+:java-docs:`deleteOne() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteOne(org.bson.conversions.Bson)>` 
+or :java-docs:`deleteMany() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteMany(org.bson.conversions.Bson)>` methods.
+
+The ``deleteOne()`` method deletes a single document. If the query
+filter matches more than one document, the method will remove the first
+occurrence of a match in the collection. 
+
+The ``deleteMany()`` method deletes all documents that match the query
+filter.
+
+Examples
+--------
+
+The following examples are about a paint store that sells eight different
+colors of paint. The store had their annual online sale resulting in the
+following documents in their ``paint_inventory`` collection: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 5 }
+    { "_id": 2, "color": "purple", "qty": 8 }
+    { "_id": 3, "color": "blue", "qty": 0 }
+    { "_id": 4, "color": "white", "qty": 0 }
+    { "_id": 5, "color": "yellow", "qty": 6 }
+    { "_id": 6, "color": "pink", "qty": 0 }
+    { "_id": 7, "color": "green", "qty": 0 }
+    { "_id": 8, "color": "black", "qty": 8 }
+
+Using ``deleteMany()``:
+```````````````````````
+
+All the paint colors in the collection are visible on their site. The
+owner would like to change the site to show colors in stock (``qty`` is
+not ``0``).  
```

Comment:
```suggestion
The paint store website displays all documents in the ``paint_inventory``
collection. To reduce customer confusion, any colors that are out of stock
should be removed.  
```

Just a quick stab at this. It can probably be improved upon.